biogrid/libs/grid-simulator/src/lib/measurements/index.ts

```diff
@@ -1,2 +1,10 @@
 export type Distance = number;
 export type Energy = number;
+
+/**
+ * Position from the top left corner of a town
+ */
+export interface ItemPosition {
+  x: Distance;
+  y: Distance;
```

Comment:
Can we make the position x and y readonly so that they are not modified at any point after setting them